logstash-core/lib/logstash/agent.rb

```diff
@@ -3,12 +3,16 @@
 require "logstash/environment"
 require "logstash/errors"
 require "logstash/config/cpu_core_strategy"
+require "stud/trap"
+require "logstash/config/loader"
 require "uri"
 require "net/http"
 require "logstash/pipeline"
-LogStash::Environment.load_locale!
```

Comment:
I didn't see a need for it (didn't break anything). we still do `load_locale!` in 3 places:

logstash-core/lib/logstash/config/mixin.rb:LogStash::Environment.load_locale!
logstash-core/lib/logstash/environment.rb:    def load_locale!
logstash-core/lib/logstash/runner.rb:LogStash::Environment.load_locale!

Which should cover the paths to loading agent.rb